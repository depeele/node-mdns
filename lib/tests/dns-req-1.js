var Util    = require('util'),
    Message = require('../message.js'),

    data    = [ // Header
                0x00, 0x02, 0x01, 0x00,
                0x00, 0x01, 0x00, 0x00,
                0x00, 0x00, 0x00, 0x00,

                // Message
                0x04, 0x70, 0x6F, 0x70,
                0x64, 0x02, 0x69, 0x78,
                0x06, 0x6E, 0x65, 0x74,
                0x63, 0x6F, 0x6D, 0x03,
                0x63, 0x6F, 0x6D, 0x00,
                0x00, 0x01, 0x00, 0x01 ],
    buf     = new Buffer( data ),
    message = new Message( buf );

console.log("message:\n%s", Util.inspect(message, false, 20));
