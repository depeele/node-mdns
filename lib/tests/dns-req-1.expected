message:
{ header: 
   { id: 2,
     qr: 0,
     opcode: 0,
     aa: 0,
     tc: 0,
     rd: 1,
     ra: 0,
     z: 0,
     ad: 0,
     cd: 0,
     rcode: 0,
     qdCount: 1,
     anCount: 0,
     nsCount: 0,
     arCount: 0,
     length: 12 },
  question: [ { qname: 'popd.ix.netcom.com', qtype: 1, qclass: 1, length: 24 } ],
  answer: [],
  authority: [],
  additional: [],
  length: 36 }
